static/js/settings_users.js

```diff
@@ -505,19 +506,27 @@ function handle_reactivation($tbody, $status_field) {
         const $button_elem = $(e.target);
         const $row = $button_elem.closest(".user_row");
         const user_id = Number.parseInt($row.attr("data-user-id"), 10);
-        const url = "/json/users/" + encodeURIComponent(user_id) + "/reactivate";
-        const data = {};
 
-        const opts = {
-            success_continuation() {
-                update_view_on_reactivate($row);
-            },
-            error_continuation(xhr) {
-                ui_report.generic_row_button_error(xhr, $button_elem);
-            },
-        };
+        function handle_confirm() {
+            const $row = get_user_info_row(user_id);
+            const $row_reactivate_button = $row.find("button.reactivate");
+            $row_reactivate_button.prop("disabled", true).text($t({defaultMessage: "Working…"}));
```

Comment:
Is this a loading indicator style we currently use? I thought we generally preferred to turn the button into a loading spinner (see e.g. message edit).

Reply:
Yes, for deactivate button we use the same. I thought it would be consistent so used the same as deactivating button.

https://github.com/zulip/zulip/blob/af5d0d6f5e5444332f9f8e565d97f4acdceaa72f/static/js/settings_users.js#L448-L451

Should I still turn the button into a loading spinner?

Reply:
> Is this a loading indicator style we currently use? I thought we generally preferred to turn the button into a loading spinner (see e.g. message edit).

We show the spinner in button when deactivating user from `Manage user` modal opened from user info popover.